assessments/guide/README.md

```diff
@@ -4,46 +4,118 @@ To provide the CNCF’s TOC with effective information about the security of
+This presentation should go over the self-assessment and provide SIG-Security
+with an initial understanding of the project.  It is recommended the **project
+lead** submit the issue as the primary point of contact (POC).
 
-Note: Responsible roles for specific items are in **bold**.
+#### Present the project and self-assessment
 
-### 1. [Create tracking issue](https://github.com/cncf/sig-security/issues/new?assignees=&labels=assessment&template=security-assessment.md&title=%5BAssessment%5D+Project+Name)
-The tracking issue serves to initiate the security assessment process.  It
-provides an initial set of information to assist SIG-Security prioritze the
-security assessment as well as provide potential reviewers with a central
-location to manage the assessment.
+Be sure to add the presentation to proposed agenda topics in the [meeting
+notes](https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g/)
+and include the POC or **project lead**.  The community may provide feedback on the self-assessment or
+ask questions about the project.  Include anything you feel important in an
+updated self-assessment based on feedback and discussion.
+
+#### Submit a PR to include the self-assessment in the repo
+
+After the presentation, the **project lead** or their designee should submit a PR, 
+citing the presentation issue number to add the self-assessment to [assessments/projects](https://github.com/cncf/sig-security/tree/master/assessments/projects) under its 
+own folder.  The ticket may then be closed after merged in.
+
+### Incubating projects
+
+Note: Responsible roles for specific items are in **bold**. If an incubation
+project did not complete a self-assessment during sandbox, they are recommended
+to start with the self-assessment before pursing joint evaluation.
+
+#### [Create tracking issue](https://github.com/cncf/sig-security/issues/new?assignees=&labels=assessment&template=security-assessment.md&title=%5BAssessment%5D+Project+Name)
+
+The tracking issue serves to initiate the joint-evaluations.  It provides
+ an initial set of information to assist SIG-Security in  prioritizing the
+joint evaluation as well as provide potential reviewers with a central
+location to manage the effort.
    * Issue may be a request from **TOC liason** or **project** itself 
-### 2. Project provides self-assessment & reviewers are assigned 
-The project performs a self-assessment using the [outline](outline.md) to
-provide the reviewers with security relevant information about their project.
-The self-assessment can include links to external documents and sources within
-the projects repository to provide additional details or reference where a
-process is kept.
-   * **[Project lead](project-lead.md)** responds to the issue with draft
-     document (see [outline](outline.md))
-   * Issue assigned to **lead [security reviewer](security-reviewer.md)** who
+   * **Security evaluation facilitator** with help from the **technical leads**
```

Comment:
can we link to security evaluation facilitator

Reply:
I think we may have to update this in the README.md and governance.md as well

Reply:
@lumjjb which update?  content changes or the link to the security evaluation facillitator role?